src/pages/DataAccessRequestApplication_new.js

```diff
@@ -1,22 +1,21 @@
 import { Component } from 'react';
```

Comment:
there's still a lot of cleanup to do to get the new DAR form functional + well designed, but I did my best to incorporate the new step 2 section into the old code.

Reply:
I will probably do the cleanup needed to get this to properly submit & look good as part of finishing up https://github.com/DataBiosphere/duos-ui/pull/1805